google/cloud/aiplatform/experimental/vertex_model/base.py

```diff
@@ -379,10 +452,15 @@ class VertexModel(metaclass=abc.ABCMeta):
     dependencies = [
         "pandas>=1.3",
         "torch>=1.7",
-        "google-cloud-aiplatform @ git+https://github.com/googleapis/python-aiplatform@refs/pull/659/head#egg=google-cloud-aiplatform",
+        GITHUB_DEPENDENCY,
     ]
 
+    container_uri = "us-docker.pkg.dev/vertex-ai/training/scikit-learn-cpu.0-23:latest"
+
     machine_type = "n1-standard-4"
+    replica_count = 1
```

Comment:
These values are class properties but they are being used as instance properties in the README. I would recommend setting these in the constructor below.

Reply:
sounds good!